src/sub-commands/flex-plugin.ts

```diff
@@ -174,7 +174,9 @@ export default class FlexPlugin extends baseCommands.TwilioClientCommand {
     await super.run();
 
     if (this.opts.runInDirectory && !this.isPluginFolder()) {
-      throw new TwilioCliError(`${this.cwd} directory is not a flex plugin directory.`);
+      throw new TwilioCliError(
+        `${this.cwd} directory is not a flex plugin directory. You must either run a plugin inside a directory or use the --name flag`,
```

Comment:
`${this.cwd} directory is not a flex plugin directory. You must either run a plugin inside a plugin directory or set the plugin by using the --name flag.`